### PR TITLE
Added CMake options to README

### PR DESCRIPTION
**Pull request by justinboswell**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,12 @@ Or on windows,
     cmake -G "Visual Studio 14 2015 Win64" ../aws-c-common
     msbuild.exe ALL_BUILD.vcproj
 
+### CMake Options
+* -DCMAKE_CLANG_TIDY=/path/to/clang-tidy (or just clang-tidy or clang-tidy-7.0 if it is in your PATH)
+* -DENABLE_SANITIZERS=ON - Enables gcc/clang sanitizers, by default 'address' and 'undefined'
```
Comment (graebm) [inline README.md]:
```
by default 'address' and 'undefined'
```
wut?

Comment (ColdenCullen) [reply README.md]: -fsanitize=address,undefined

Comment (JonathanHenson) [reply README.md]: yues

File: README.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ Or on windows,
     msbuild.exe ALL_BUILD.vcproj
 
 ### CMake Options
-* -DCMAKE_CLANG_TIDY=/path/to/clang-tidy (or just clang-tidy or clang-tidy-7.0 if it is in your PATH)
-* -DENABLE_SANITIZERS=ON - Enables gcc/clang sanitizers, by default 'address' and 'undefined'
+* -DCMAKE_CLANG_TIDY=/path/to/clang-tidy (or just clang-tidy or clang-tidy-7.0 if it is in your PATH) - Runs clang-tidy as part of your build.
+* -DENABLE_SANITIZERS=ON - Enables gcc/clang sanitizers, by default this adds -fsanitizer=address,undefined to the compile flags.
```
Comment (ColdenCullen) [inline README.md]: * for projects that call aws_add_sanitizers

**Pull request merged**